build-repo.sh

```diff
@@ -43,6 +43,12 @@ esac
 
 # Setup environment variables
 
+if [ ! -z "${JOBS}" ]; then
+  sudo sed -i "s/export JOBS=.*/export JOBS=${JOBS}/" /etc/abuild.conf
+fi
+
+sudo sed -i 's/export MAKEFLAGS=-j$JOBS/export MAKEFLAGS="-j$JOBS -l$JOBS"/' /etc/abuild.conf
```

Comment:
As the default MAKEFLAGS can be changed by future updates of abuild, match regex is better to be more generic. (For example, if `""` is added, it doesn't match)
```suggestion
sudo sed -i 's/export MAKEFLAGS=.*/export MAKEFLAGS="-j$JOBS -l$JOBS"/' /etc/abuild.conf
```